samples/train_metrics.json

```diff
@@ -0,0 +1,7 @@
+{
+    "train_metrics": {
+        "mse": 580148.0536563194,
+        "mae": 567.6535461416039,
+        "rmse": 761.6745063715337
```

Comment:
How do we take multidimensional metrics?

Reply:
added sample in the file